packages/ui/src/app/pages/Proposals/CurrentProposals.stories.tsx

```diff
@@ -1405,7 +1405,24 @@ export const SpecificParametersSetMembershipPrice: Story = {
     })
   }),
 }
+export const SpecificParametersUpdatePalletFrozenStatus: Story = {
+  play: specificParametersTest('Update Pallet Frozen Status', async ({ args, createProposal, modal, step }) => {
+    await createProposal(async () => {
+      const nextButton = getButtonByText(modal, 'Create proposal')
+      expect(nextButton).toBeDisabled()
 
+      await userEvent.click(modal.getByTestId('crt-feature-select'))
```

Comment:
After you have addressed the other comments, this test should make sure the warning and validation error appear, then it can click again to make the form valid and continue